README.md

```diff
@@ -39,37 +39,34 @@ accuracy and reliability of your generative AI models, [read this
 blog](https://developer.nvidia.com/blog/enhancing-rag-applications-with-nvidia-nim/).
 
 Get started with NIM Anywhere now with the [quick-start](#quick-start)
-instructions and build your first RAG application using NIMs\!
+instructions and build your first RAG application using NIMs!
```

Comment:
```suggestion
instructions and build your first RAG application using NIM Microservices!
```

Reply:
NIMs are Nvidia Inference Microservices.  So an NIM Microservice is a Nvidia Inference Microservice Microservice 

Kind of like ATM Machine would be wrong.

Reply:
Following the convention seen out [here](https://www.nvidia.com/en-eu/ai/?ncid=pa-srch-goog-201921&_bt=728997781338&_bk=nvidia%20ai&_bm=b&_bn=g&_bg=173378669933&gad_source=1&gclid=EAIaIQobChMIpfqSruSViwMVXlz_AR2WbA48EAAYASAAEgKMfvD_BwE) and present in other updated public-facing content.